hutch_python/load_conf.py

```diff
@@ -228,8 +228,10 @@ def load_conf(conf, hutch_dir=None):
     if proposal is not None:
         qs_objs = get_qs_objs(proposal, run)
         cache(**qs_objs)
-        exp_objs = get_exp_objs(proposal, run)
-        cache(**exp_objs)
+        user = get_exp_objs(proposal, run)
+        for name, obj in qs_objs.items():
+            setattr(user, name, obj)
```

Comment:
Should we at all be concerned about us getting a `User` object we can't set attributes on? Or I'm just being paranoid.

Reply:
I didn't consider this. How would this happen?

Reply:
If someone made object inherit from some strange object. Probably not worth worrying about.

Reply:
I suppose I can imagine something a little less malicious, like:

```
@property
def inj_x(self):
    return something
```

But I think for these instances we simply say... don't do that!